maas/plugins/nova_cloud_stats.py

```diff
@@ -106,6 +111,20 @@ def main(args):
     with print_output():
         parser = argparse.ArgumentParser(
             description='Check Nova hypervisor stats')
+        parser.add_argument('--cpu',
+                            type=int,
```

Comment:
I think this is the cause of the failure. You're supplying floats but requiring ints.